analyses/upstream-analysis/README.md

```diff
@@ -0,0 +1,116 @@
+  
+Here, the user can select to implement the following strategies to remove low quality cells:
+- `step A` [miQC](https://bioconductor.org/packages/devel/bioc/vignettes/miQC/inst/doc/miQC.html) R package. The miQC model is based on the assumption that there are a non-trivial number of compromised cells in the dataset, which is not true in all datasets. If it is already known that, we recommend that the user skip this step. 
+- `step B` `run_QC_default` function. This is split in two filtering steps.
+   - `step 1`: Filter cells with low content of genes expressed and remove mtDNA from each library (as defined in the `params`).
+   - `step 2`: `Find_Outlier_Thershold` function. This is an optional step (as defined in the `params`). 
+If miQC does not identify any low quality cells, then the `step B` will automatically be used as for the filtering strategy.
+
+#### Post alignment/cell quality filtering parameters
+We recommend that the user use the following parameters for initial QC, and then adjust accordingly if necessary:
+- `scRNA`: min_genes = 300 (nFeature_RNA (genes detected))
+           min_count = 500 (nCount_RNA (UMIs detected))
+           mtDNA_pct_default = 15 (ideally 10; Percent Mitochondrial)
+- `snRNA`: min_genes = 300 (nFeature_RNA (genes detected))
+           min_count = 500 (nCount_RNA (UMIs detected))
+           mtDNA_pct_default = 5 (ideally 1; Percent Mitochondrial)
+
+### (2) Estimating and filtering out ambient mRNA (`empty droplets`)
+
+[SoupX](https://cran.r-project.org/web/packages/SoupX/vignettes/pbmcTutorial.html) profiles “the soup”, i.e., collection of cell-free mRNAs floating in the input solution. The soup looks different for each input solution and strongly resembles the expression pattern obtained by summing all the individual cells.
+
+SoupX calculates `Cell-specific contamination fraction` (Estimate (or manually set) the contamination fraction, the fraction of UMIs originating from the background, in each cell) and infers a `corrected expression matrix` (Correct the expression of each cell using the ambient mRNA expression profile and estimated contamination).
+
+The CellRanger output from the `cellranger-analysis` module is used for this step.
+ -  Contamination summary table and Cell-specific contamination fraction plot are generated.
+
+
+### (3) Estimating and filtering out doublets
```

Comment:
This question is more so for my own understanding, but is the doublet filtering always automatically applied or is it also optional?

Reply:
For now, it is not optional, doublets will always be filtered out at the last step. I can make it optional if that is necessary.

Reply:
@AntoniaChroni I think it may be a good idea to have it be optional. My understanding is that the core often has not done doublet filtering for projects involving neuronal cells because expression patterns of different cell types can overlap or be on a spectrum of expression that can make it difficult for doublets to be identified correctly sometimes. If it were possible, I think the best option would be to have doublet results generated but maybe not automatically filtered?

Reply:
We can discuss this when we get to that script and data analysis. `SoupX` is optional, so we could do the same for doublets. We can edit the `README` accordingly at that time.